Makefile

```diff
@@ -11,8 +11,6 @@ package_name=$(app_name)
 cert_dir=$(HOME)/.nextcloud/certificates
 version+=master
 
-all: appstore
```

Comment:
@nickvergessen there was a duplicate all key here.
Can you check the makefile? I'm not sure it still fits this app :)

Reply:
looks good, for packaging I do `make release` always anyway